lib/cfa/augeas_parser.rb

```diff
@@ -19,6 +36,7 @@ def add(value, placer = AppendPlacer.new)
       element = placer.new_element(@tree)
       element[:key] = augeas_name
       element[:value] = value
+      # FIXME: load_collection missing here
```

Comment:
I did not find AugeasCollection being actually used anywhere, so it seems it is an unfinished and untested concept.